org-formation/organization.yaml

```diff
@@ -248,3 +249,13 @@ Organization:
       Tags:
         Department: Platform
         Project: Infrastructure
+
+  NlpSandboxAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: NlpSandboxAccount
+      RootEmail: support@nlpsandbox.io
+      Alias: org-sagebase-nlpsandbox
+      Tags:
+        Department: CompOnc
+        Project: Infrastructure
```

Comment:
This project name is probably not correct... should is be `Challenges` or `NLP-Challenge`?